internal/node/launcher.sh

```diff
@@ -171,10 +171,9 @@ if [[ "${BAZEL_NODE_PATCH_REQUIRE}" != /* ]] && [[ ! "${BAZEL_NODE_PATCH_REQUIRE
   export BAZEL_NODE_PATCH_REQUIRE=$(pwd)/${BAZEL_NODE_PATCH_REQUIRE}
 fi
 
-readonly repository_args=$(rlocation "TEMPLATED_repository_args")
 readonly lcov_merger_script=$(rlocation "TEMPLATED_lcov_merger_script")
 
-source $repository_args
+export NODE_REPOSITORY_ARGS="--preserve-symlinks"
```

Comment:
I had a look this time where this actually comes from, its from [here](https://github.com/Canva/rules_nodejs/blob/97ad57a9324dcdbff495e20683029358023ac66c/internal/node/node_repositories.bzl#L468). Might be worth adding a comment along the lines of

```suggestion
# Assumes "preserve_symlinks = False" wasn't passed to "node_repositories()" in WORKSPACE (defaults to True)
# We hard code this here because it otherwise comes from a host-platform specific
# path like nodejs_darwin_arm64/bin/node_repo_args.sh which breaks cache sharing
# between host!=target and host==target builds.
export NODE_REPOSITORY_ARGS="--preserve-symlinks"
```

Reply:
removed wholesale in fe88e5c3